### ISSUE TITLE
--Help Arg not printing

### ISSUE THREAD

**Issue by DarkDot** [vovoid/vsxu#219]
Using the --help arg doesn't print the avaible args, It causes powershell and cmd to close without launching VSXu player

**Comment by DarkDot** [comment]
Looks like it requires the -console arg.

**Comment by jaw** [comment]
Yes, a peculiar thing about how windows handles its terminal. One would 
think an application would have no right to close a command window it 
didn't open itself but it can...
